.github/workflows/canary-build.yml

```diff
@@ -3,45 +3,59 @@ on:
   push:
     branches:
       - canary
+    # only run on changes to these files
+    paths:
+      - "**.js"
+      - "**.jsx"
```

Comment:
Same as above, we should check for `.scss` files too

Reply:
updated